llama-index-core/llama_index/core/settings.py

```diff
@@ -113,11 +113,11 @@ def tokenizer(self) -> Callable[[str], List[Any]]:
         """Get the tokenizer."""
         import llama_index.core
 
-        if llama_index.global_tokenizer is None:
+        if llama_index.core.global_tokenizer is None:
```

Comment:
I could've sworn we've fixed this before. I remember doing search for global_tokenizer specifically. Maybe got overwritten?

Reply:
it might have gotten overwritten at some point lol

Reply:
lots of moving parts lately

Reply:
Lol yah that's for sure. Oh well -- we do our best!